sos/plugins/composer.py

```diff
@@ -10,14 +10,14 @@ class Composer(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
 
     packages = ('composer-cli',)
 
-    def _get_blueprints(self):
-        blueprints = []
-        bp_file = self.get_cmd_output_now("composer-cli blueprints list")
+    def _get_entries(self, cmd):
+        entries = []
+        bp_file = self.get_cmd_output_now(cmd)
```

Comment:
Worth renaming "bp_file" to "entries_file" or similar? "bp_" prefix is confusing / refers to the previous "blueprint".

Otherwise +1 to the commit/PR.

Reply:
Updated the PR. Originally thought it wasn't, but on a second read yeah it was a bit confusing.